src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -1,4 +1,400 @@
 privacy_notices:
+### GPP State-by-state ###
+- name: Data sales and sharing
+  notice_key: data_sales_sharing_gpp_us_state
+  id: pri_gpp_1 # TODO: update
```

Comment:
updated mine in 923c532b1af30c12e2aca8ebd8386c7d35fda9b8 to be more similar to existing ids. i put a suffix that's bot a bit more than a few letters from the notice name since that's pretty ambiguous now (we have similar/overlapping names). the suffixes indicate GPP and us nat vs. state. i think that should be fine - let me know if it's problematic or seems odd in any way.